rdrf/rdrf/forms/navigation/quick_links.py

```diff
@@ -350,32 +308,32 @@ def _build_menu(self):
                 **Links.QUESTIONNAIRE,
             }
 
-            MenuConfig().genetic_staff = {
+            self.menu_config.genetic_staff = {
                 **Links.DATA_ENTRY
             }
 
-            MenuConfig().genetic_curator = {
+            self.menu_config.genetic_curator = {
                 **Links.DATA_ENTRY,
                 **Links.GENETIC
             }
 
-            MenuConfig().clinical = {
+            self.menu_config.clinical = {
                 **Links.DATA_ENTRY,
                 **Links.QUESTIONNAIRE,
                 **Links.VERIFICATION,
             }
 
             # Super user has combined menu of all other users
```

Comment:
Looks like a strange idea to me.
ie. that adding up menus of users with different roles would necessarily add up to all the menus in the system...